test/featureUtils_unit_test.js

```diff
@@ -0,0 +1,79 @@
+import Fetcher from '../src/Core/Scheduler/Providers/Fetcher';
+import GeoJSON2Feature from '../src/Renderer/ThreeExtended/GeoJSON2Feature';
+import FeaturesUtils from '../src/Renderer/ThreeExtended/FeaturesUtils';
+import Coordinates from '../src/Core/Geographic/Coordinates';
+/* global describe, it */
+
+const assert = require('assert');
+const fs = require('fs');
+
+const counters = {
+    fetch: [],
+};
+
+global.fetch = function _fetch(url) {
+    var res;
+    var pr = new Promise((resolve) => {
+        res = resolve;
+    });
+    // try reading as a file
+    fs.readFile(`${process.env.PWD}/examples/${url}`, 'utf-8', (err, content) => {
+        if (!content || err) {
+            counters.fetch.push(url);
+        }
+        res({
+            json: () => Promise.resolve(JSON.parse(content)),
+            ok: true,
+        });
+    });
+    return pr;
+};
+
+const promiseGeoJson = Fetcher.json('./geojson/simple.geojson').then(function _(result) {
```

Comment:
No need to use Fetcher (and mock fetch): juste use `fs.readFile` + `JSON.parse` directly.

Reply:
Or even simpler : `const json = require('./geojson/simple.geojson');` should work...

Reply:
update